content/en/ddsql_editor/reference/expressions_and_operators.md

```diff
@@ -39,20 +39,27 @@ DDSQL implements the following comparison operators:
 
 For tag references and tag groups, the equality operator (`=`) is treated as a "contains" comparison. See the [Querying Tags in DDSQL][1] for more details.
 
-Additionally, DDSQL supports the following SQL keywords, which function as standard boolean operators:
+## SQL comparison keywords
 
-- `NOT`
-- `AND`
-- `OR`
+DDSQL supports the following SQL keywords, which function as standard Boolean operators:
+
+| Operator | Description            | Example | Result |
+|----------|------------------------|---------|--------|
+| `NOT`    | Filter records based on more than one condition. | `SELECT * FROM Student WHERE NOT Major = 'Economics';`   | Return all students that do not major in Economics.  |
+| `AND`    | Filter records based on more than one condition. | `SELECT * FROM Student WHERE Age = 18 AND Major = 'Economics';`   | Return all students that are age 18 AND major in Economics.  |
+| `OR`     | Filter records based on more than one condition. | `SELECT * FROM Student WHERE Age = 18 OR Major = 'Economics';`   | Return all students that are either age 18 OR major in Economics.  |
 
 DDSQL also supports the following comparator keywords as they are defined in the SQL standard:
 
-- `IS NULL`
-- `IS NOT NULL`
-- `LIKE`
-- `NOT LIKE`
-- `IN`
-- `NOT IN`
+| Operator     | Description            | Example | Result |
+|--------------|------------------------|---------|--------|
+| `IS NULL`    | Select rows if the specified field is null. | `SELECT * FROM albums WHERE artist IS NULL`   | Return all rows that contain no data in the `artist` column.  |
+| `IS NOT NULL`| Select rows if the specified field is not null. Exclude rows with missing data. | SELECT * FROM albums WHERE artist IS NOT NULL | Return all rows that contain data in the `artist` column.   |
+| `LIKE`       | Search for a specific pattern in a string value. You can use the following wildcard characters to define the patterns: <br>**Percent sign (%)**: Represents zero, one, or multiple characters. <br>**Underscore (_)**: Represents a single character. | `SELECT * FROM employees WHERE firstname LIKE 'a%';` | Return all rows from the `employees` table where the `firstname` column value starts with `a`.  |
+| `NOT LIKE`   | Exclude rows from a search, where the row has a specific pattern in a string value. You can use the wildcards `%` and `_` for pattern matching. | `SELECT * FROM products WHERE product_code NOT LIKE '_B%';` | Return all rows from the `products` table where the `product_code` does **not** have `B` as the second character. |
```

Comment:
SELECT *
FROM aws_eks_cluster
WHERE LOWER(logging) NOT LIKE '%"enabled":true%'

Reply:
```suggestion
| `NOT LIKE`   | Exclude rows from a search, where the row has a specific pattern in a string value. You can use the wildcards `%` and `_` for pattern matching. | `SELECT * FROM aws_eks_cluster WHERE WHERE LOWER(logging) NOT LIKE '%"enabled":true%'` | Return all rows from the `aws_eks_cluster` table where the `logging` does **not** have `"enabled":true%'`. |
```